_docs/developer/getting_started/vm_install_using_vagrant.md

```diff
@@ -46,212 +52,475 @@ If you using an Intel-based Mac, you will follow the instructions below._
+        home
+        └── myusername
+            └── Submitty
+                │        GIT CHECKOUT
+                ├── Submitty       (main repository)
+                ├── AnalysisTools  (optional)
+                ├── Lichen         (optional)
+                ├── RainbowGrades  (optional)
+                └── Tutorial       (optional)
+        ```
+
+      _This host directory structure will be shared / synced between
+      your host operating system and the Submitty virtual machine._
+
+   <br>
+
+6. <a name="vagrant_up_instructions"></a> Navigate into the Submitty
+repository on your computer. If on Windows, you should run CMD or
+Powershell on administrator mode.
+
+   <br>
+
+7. Decide whether your development work will *require sample
+assignment submissions or autograding results.* If you do not need neither, then type ``NO_SUBMISSIONS=1`` but do **not press enter
+just yet** if on MacOS or Linux. The current command will be
+extended in the next step.
+
+   However, _if you are on Windows_, you actually **must** press
+   enter on``NO_SUBMISSIONS=1`` before proceeding, rather than
+   waiting to extend the command as MacOS and Linux users will do.
+
+   **Again, only enter ``NO_SUBMISSIONS=1`` if you do not need
+   autograding.**
+
+   Skipping the creation of the sample submissions and their
+   autograding will decrease the time to complete installation,
+   so this tradeoff may be useful to some.
+
+   <br>
+   
+   >##### Turn on Autograding after Disabling
+   >If you decide you do want autograding later on, simply type
+   >
+   >  ```
+   >  SET NO_SUBMISSIONS=
+   >  ```
+   >
+   >to unset the environment variable. Alternatively, the
+   >environment variable will reset to off upon a new terminal
+   >session anyways, as dicussed in the [section about starting up your VM again](#vagrant_up_again) after shutdown. You may
+   >check whether the environment variable is set or not by typing
+   >
+   >  ```
+   >  SET NO_SUBMISSIONS
+   >  ```
+
+   <br>
+
+8. In the same directory as the previous step, type:
 
-   7. Enter your **BIOS** (generally by pressing Del, F12, or other keys while booting). If you are not able to find the key combo needed to enter your BIOS, refer to [this guide](https://www.tomshardware.com/reviews/bios-keys-to-access-your-firmware,5732.html).
+   ```
+   vagrant up
+   ```
 
-   8. Locate **Virtualization**, and enable it. (Note: If you cannot find the option to enable virtualization, [search Google](http://tinyurl.com/enable-virtualization) for a tutorial on enabling it with your motherboard.)
+   Vagrant will build your VM.  This will take maybe 30 minutes to a
+   few hours depending on your Internet connection speed.  When this
+   command finishes, your VM is ready to use.
 
-   9. Reboot your computer.
+---
 
-   **Ubuntu**
-   1. Enter your **BIOS** (generally by pressing Del, F12, or other keys while booting).
+## <a name="vagrant-up-troubleshooting"> </a> Vagrant Up Troubleshooting
```

Comment:
You don't need the `a` tag here either, markdown rendering in Jekyll will generate HTML with built in anchor tags with the `##` syntax automatically